applications/nrf_desktop/configuration/nrf52_pca20037/hid_keymap_def.h

```diff
@@ -74,7 +74,7 @@ static const struct hid_keymap hid_keymap[] = {
 	{ KEY_ID(0x03, 0x0C), 0x62, IN_REPORT_KEYBOARD_KEYS }, /* keypad 0 */
 	{ KEY_ID(0x03, 0x0D), 0x63, IN_REPORT_KEYBOARD_KEYS }, /* keypad . */
 	{ KEY_ID(0x03, 0x0E), 0x50, IN_REPORT_KEYBOARD_KEYS }, /* arrow left */
-	{ KEY_ID(0x03, 0x11), 0x81, IN_REPORT_KEYBOARD_KEYS }, /* volume down */
+	{ KEY_ID(0x03, 0x11), 0x01, IN_REPORT_CONSUMER_CTRL }, /* volume down */
```

Comment:
Using bit position instead of usage id seems problematic for me as now the information placed does not match what the structure member is suppose to hold. There is no explanation in the comment how does 0x01 refer to the volume down.
Note that in other cases value placed under the usage id matches with what USB HID tables define.

I know this is done to make the code neater and faster but I am wondering if we are not paying too much in terms of configurability of our code.

Reply:
Let's discuss this further offline